examples/dfem/minimal_surface.cpp

```diff
@@ -0,0 +1,518 @@
+//                       MFEM Example - Minimal Surface
+//
+// Compile with: make minimal_surface
+//
+// Sample runs:  mpirun -np 4 minimal_surface -der 0
+//               mpirun -np 4 minimal_surface -der 0 -o 2
+//               mpirun -np 4 minimal_surface -der 0 -r 2
+//               mpirun -np 4 minimal_surface -der 1
+//               mpirun -np 4 minimal_surface -der 2
```

Comment:
This sample run fails on my system when the CGSolver computes a NaN. Adding either `-r 1` or `-o 2` corrects this but perhaps another solution would be better.

Reply:
That's expected since the initial mesh is just one element... I'll fix it.